package.json

```diff
@@ -4,10 +4,13 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "bos-workspace dev",
-    "build": "bos-workspace build"
+    "bw": "bos-workspace",
+    "dev": "bw ws dev",
```

Comment:
Since we're downgrading from a "workspace" to an "app", we can remove the "ws" (workspace) from all our scripts.